frontend/ui/widget/notification.lua

```diff
@@ -93,6 +101,25 @@ function Notification:init()
     }
 end
 
+function Notification:setNotificationSource(source)
+    self.source = source
+end
+
+function Notification:setNotificationVerbosity(requested_verbosity)
+    self.verbosity = requested_verbosity
+end
+
+function Notification:notification(text, requested_level)
```

Comment:
displaySomethingSomething or just display would read better, wouldn't it?

Reply:
`Notification:notify(text...)` , as an action.

Reply:
And given that this Notification object would now also work as a Notification manager, may be use "notify" in all the method names related to its use as a notification manager:
```lua
:notify()
:setNotifySource()
```